### PR TITLE
W2 D3 curriculum specialist & day lead updates 

### PR DESCRIPTION
**Pull request by nsea-log**
New T1 to T4 and T6, T5 included but needs tbd.

### CODE REVIEW

**Review by iamzoltan — CHANGES_REQUESTED** [review]
Great work @nsea-log !! Just a few minor tweaks.

- [ ]  Look into the odd metadata behaviour
- [ ]  Remove the left over merge conflict text
- [ ]  Fix kernel

Really good work! 

p.s do you want to leave in the todos for now?

File: tutorials/W2D3_FutureClimate-IPCCII&IIISocio-EconomicBasis/W2D3_Tutorial4.ipynb [hunk 1]
```diff
@@ -273,1649 +241,460 @@
...
  ],
  "metadata": {
   "colab": {
    "collapsed_sections": [],
    "include_colab_link": true,
-   "name": "W2D3_Tutorial4",
+   "name": "W2D3_Tutorial2",
    "provenance": [],
    "toc_visible": true
   },
```
[the remark this answers is not in the document]
Comment (nsea-log) [reply tutorials/W2D3_FutureClimate-IPCCII&IIISocio-EconomicBasis/W2D3_Tutorial4.ipynb:700]: None.

[the remark this answers is not in the document]
Comment (nsea-log) [reply tutorials/W2D3_FutureClimate-IPCCII&IIISocio-EconomicBasis/W2D3_Tutorial4.ipynb:700]: Changed it manually.

**Review by iamzoltan — APPROVED** [review]
looks great!

**Pull request merged**